src/internal/proc.js

```diff
@@ -626,6 +640,18 @@ export default function proc(
     channel.flush(cb)
   }
 
+  function runGetContextEffect(prop, cb) {
+    // TODO: if object returned -> return shallow copy?
+    // should yield getContext() be available to access context?
+    // if yes then shallow copying it is a must to avoid unexpected mutations
+    cb(taskContext[prop])
```

Comment:
Ok, I tend to agree with you that throwing is not really necessary here as any relevant checks could be easily performed on the user side.